.github/workflows/register-schemas.yaml

```diff
@@ -0,0 +1,59 @@
+on:
+  push:
+    branches: master
+
+name: Register schemas & update table
+
+env:
+  RENV_PATHS_ROOT: ~/.local/share/renv
+
+jobs:
+  register-schemas-dryrun:
+    runs-on: macos-10.15
```

Comment:
Any particular reason you are using macos?

Reply:
Nope. It's just what Kara set the dryrun action up as. I just copy/pasted and updated that action.

Reply:
i believe GH, just like all other CI services, has much less macs than linux.  i'm guessing that linux is much more available and much faster